src/wireviz/wireviz.py

```diff
@@ -115,6 +116,7 @@ def parse(
         metadata=Metadata(**yaml_data.get("metadata", {})),
         options=Options(**yaml_data.get("options", {})),
         tweak=Tweak(**yaml_data.get("tweak", {})),
+        source_path=source_path
```

Comment:
I suggest reusing the existing `image_paths` that already has collected all input folders:
```suggestion
        input_folders=image_paths,
```